build/get-changelogs.sh

```diff
@@ -0,0 +1,26 @@
+# Generates the changelog (into `JSON`) for each rule, under `_rules/*.md` directory.
+
+# iterate every rule file
+for filepath in _rules/*.md
+do
+  # get rule filename
+  filename=$(basename "${filepath%.*}")
+
+  # parse rule `id` from `filename`
+  id=${filename##*-}
+
+  # create `changelogs` directory under `_data/`
+  mkdir -p _data/changelogs
```

Comment:
Note to self: 
- clean `_data/changelogs` outside the loop
- try and use a node script instead of a shell script (Ensure windows compatibility).